docs/schema/aws.md

```diff
@@ -1758,3 +1760,37 @@ Representation of an AWS S3 [Bucket](https://docs.aws.amazon.com/AmazonS3/latest
         ```
         (S3Bucket)-[TAGGED]->(AWSTag)
         ```
+
+## KMSKey
+
+Representation of an AWS [KMS Key](https://docs.aws.amazon.com/kms/latest/APIReference/API_KeyListEntry.html).
+
+| Field | Description |
+|-------|-------------|
+| firstseen| Timestamp of when a sync job first discovered this node  |
+| lastupdated |  Timestamp of the last time the node was updated |
+| **id** | The id of the key|
+| name |  The name of the key |
+| description |  The description of the key |
+| enabled |  Whether the key is enabled |
+| region | The region where key is created|
+
+### Relationships
+
+- AWS KMS Keys are resources in an AWS Account.
+
+        ```
+        (AWSAccount)-[RESOURCE]->(KMSKey)
+        ```
+
+- AWS KMS Key may also be refered as KMSAlias via aliases.
+
+        ```
+        (KMSKey)-[KNOWN_AS]->(KMSAlias)
```

Comment:
Would it be possible to add an entry in the schema also for `KMSAlias` and `KMSGrant` since they are treated as separate entities?

Reply:
@marco-lancini Updated the schema documentation with KMSAlias and KMSGrant in latest commit